src/v/ssx/work_queue.h

```diff
@@ -52,6 +63,7 @@ class work_queue {
     error_reporter_fn _error_reporter;
     ss::condition_variable _cond_var;
     ss::chunked_fifo<ss::noncopyable_function<ss::future<>()>> _tasks;
+    is_running_t _is_running{is_running_t::yes};
```

Comment:
nit: consider changing this to `paused` ? as running is the default state for the queue